include/sentry.h

```diff
@@ -2058,6 +2075,94 @@ SENTRY_EXPERIMENTAL_API const char *sentry_sdk_name(void);
  */
 SENTRY_EXPERIMENTAL_API const char *sentry_sdk_user_agent(void);
 
+/**
+ * A sentry Metric.
+ *
+ * See https://develop.sentry.dev/delightful-developer-metrics/
+ */
+struct sentry_metric_s;
+typedef struct sentry_metric_s sentry_metric_t;
+
+/**
+ * Creates a counter metric.
+ *
+ * Counters are one of the more basic types of metrics and can be
+ * used to count certain event occurrences.
+ *
+ * Counter value can only be incremented.
+ */
+SENTRY_EXPERIMENTAL_API sentry_metric_t *sentry_metrics_new_increment(
+    const char *key, double value);
+SENTRY_EXPERIMENTAL_API sentry_metric_t *sentry_metrics_new_increment_n(
+    const char *key, size_t key_len, double value);
```

Comment:
Apparently that's just how counter metric type should be represented in the API according to [docs](https://develop.sentry.dev/sdk/metrics/#counters). Other SDKs seem to follow the same naming scheme. Yet I agree that this mismatch is somewhat confusing.

Reply:
Okay, then, let's not go against the grain. I hope it is acceptable that I still find this funny (given that the doc link ends with `#counter` and has the header `Counter`). It is probably to avoid a name collision.